docs/competitive-programming/max-number-from-array.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Maximum number formed from numbers in array
+slug: /competitive-programming/max-number-from-array
+---
+
+## Introduction
+
+To form the maximum number possible from an array of single-digit numbers, we have to sort the array in desscending order. The numbers appended in the descending order gives the maximum number.
+However, the same algorithm cannot be used for array containing both two-digit and single-digit numbers. 
+For example, if the array contains two elements, 2 and 21, then according to the algorithm used for the single digits, if we sort the array in descending order and append them, then the maximum number will be 212.
+But this is wrong. The maximum number that can be formed from this array is 221.
+Therefore, we need to use a separate algorithm.
```

Comment:
Wrap all of this in one line. Use backticks with numbers. Fix the typos

Reply:
Done